ID.me WebVerify SDK/IDmeWebVerify.h

```diff
@@ -61,17 +64,18 @@ typedef NS_ENUM(NSUInteger, IDmeWebVerifyErrorCode)
 
 /**
  Returns the User profile with the stored access token. 
+ @param scope The type of token to be used. If nil then the last token will be used
  @param webVerificationResults A block that returns an NSDictionary object and an NSError object. The verified user's profile is stored in an @c NSDictionary object as @c JSON data. If no data was returned, or an error occured, @c NSDictionary is @c nil and @c NSError returns an error code and localized description of the specific error that occured.
  */
-- (void)getUserProfileWithResult:(IDmeVerifyWebVerifyResults _Nonnull)webVerificationResults;
+- (void)getUserProfileWithScope:(NSString* _Nullable)scope andResult:(IDmeVerifyWebVerifyResults _Nonnull)webVerificationResults;
```

Comment:
I wouldn't use the same block type for getting the token and user profile due to them should have different argument list. When requesting the access token doesn't matter the profile parameter, and when requesting the user profile probably you don't want the access token.

What do you think? Maybe this is a change for a new PR